community/kernel-api/src/main/java/org/neo4j/internal/kernel/api/helpers/RelationshipDenseSelectionIterator.java

```diff
@@ -72,6 +72,10 @@ public R next()
             close();
             next = RelationshipSelections.NO_ID;
         }
+        else
```

Comment:
I'm a fan of consistent commits. Personally I would've merged this into https://github.com/neo4j/neo4j/pull/11041/commits/d141aeb00260d4cc6fc3c237e3daca809d61a0eb to aid our future selves.

Reply:
Done.